--- linphoneqt-4.1.1/CMakeLists.txt.orig	2017-07-21 15:14:18.000000000 +0200
+++ linphoneqt-4.1.1/CMakeLists.txt	2019-06-09 19:41:59.317048446 +0200
@@ -96,10 +96,11 @@
   include("${EP_bctoolbox_CONFIG_DIR}/BcToolboxConfig.cmake")
   include("${EP_belcard_CONFIG_DIR}/BelcardConfig.cmake")
 else ()
-  find_package(Linphone REQUIRED)
-  find_package(LinphoneCxx REQUIRED)
+  include(FindPkgConfig)
+  pkg_search_module(LINPHONE REQUIRED linphone)
+  pkg_search_module(LINPHONECXX REQUIRED linphone++)
   find_package(BcToolbox REQUIRED)
-  find_package(Belcard REQUIRED)
+  pkg_search_module(BELCARD REQUIRED belcard)
 endif ()
 
 set(SOURCES
